.vscode/settings.json

```diff
@@ -9,9 +9,6 @@
     	"**/.yarn": true,
     	"**/.pnp.*": true
 	},
-	"workbench.editorAssociations": {
-		"*.dmi": "imagePreview.previewEditor"
-	},
```

Comment:
??

Reply:
oops, local changes, dind't mean to commit